resources/js/components/Litter/Tags.vue

```diff
@@ -9,10 +9,11 @@
                 <!-- List of tags in each category -->
                 <span
                     v-for="tags in Object.entries(category.tags)"
-                    v-html="getTags(tags, category.category)"
-                    class="tag is-large is-info litter-tag"
-                    @click="removeTag(category.category, tags[0])"
-                />
+                    class="tag is-medium is-info litter-tag"
+                >
+                    {{$t(`litter.${category.category}.${tags[0]}`) + ': ' + tags[1]}}
+                    <button class="delete is-small" @click="removeTag(category.category, tags[0])"/>
```

Comment:
Keep `v-html` and `@click="removeTag` please

Reply:
Move `removeTag` to delete button, no need to remove when to tag.

Reply:
Why do we need to keep `v-html`? We just need text inside the tag.

Reply:
We want to click tag to delete it. It's UX

Reply:
We don't need to keep `-v-html` but I prefer the readability